webui/tests/tests/StepImplementation.ts

```diff
@@ -410,13 +410,15 @@ export default class StepImplementation {
 
   @Step('<action> experiment row <row>')
   public async modifyExperiment(action: string, row: string) {
-    await t.click(t.tableCell({ row: parseInt(row) + 1, col: 13 }));
+    const lastCol = (await t.$('th').elements()).length;
+    await t.click(t.tableCell({ row: parseInt(row) + 1, col: lastCol }));
```

Comment:
if the test only failed after this change (even in another spot): could we avoid addressing this by counting cols? it should work but seems more brittle than it needs to and relies on taiko. wdy about using a selector based on the dropdown component name or some other attribute inherent to the dropdown

Reply:
I like the idea, but afaik that would mean not using the `tableCell` function, and that brings up the question of how to control  which row is being selected

Reply:
ah you're right